### PR TITLE
GH Actions: Build with buildah

### PR DESCRIPTION
**Pull request by jlledom**

### CODE REVIEW

File: .github/workflows/buildah.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+name: Build dnsmasq oci container image
+
+on:
+  push:
+    branches:
+      - gh-actions
```
Comment (akostadinov) [inline .github/workflows/buildah.yml:6]: only to this branch? not main/master?

Comment (jlledom) [reply .github/workflows/buildah.yml:6]: I removed this. It was for testing. I couldn't find another way to make the action run.

Comment (akostadinov) [reply .github/workflows/buildah.yml:6]: if you have `workflow_dispatch` then you can start it manually

Comment (jlledom) [reply .github/workflows/buildah.yml:6]: I couldn't, until the job appeared for the first time in the Actions tab. And for that I had to add the push launcher.

**Pull request merged**